scheduler/src/main/scala/uk/sky/scheduler/config/Config.scala

```diff
@@ -41,13 +43,15 @@ object KafkaConfig {
         .withProperties(config.consumer.properties)
         .withAutoOffsetReset(AutoOffsetReset.Earliest)
 
-    def producerSettings[F[_] : Sync, K, V](using
+    def producerSettings[F[_], K, V](using
         Resource[F, KeySerializer[F, K]],
         Resource[F, ValueSerializer[F, V]]
     ): ProducerSettings[F, K, V] =
       ProducerSettings[F, K, V]
         .withBootstrapServers(config.producer.bootstrapServers)
         .withProperties(config.producer.properties)
+        .atLeastOnce
+        .performant
```

Comment:
Was defined but never used. Should be used to guarantee at least once semantics, and also slightly improve performance